### PR TITLE
Create new closing balance record on statement upload

### PR DESCRIPTION
**Pull request by ibrechin**
Creating the balance record on statement upload instead of
download ensures that it is unaffected by the order in which
files are downloaded.

### CODE REVIEW

File: mtp_transaction_uploader/upload.py [hunk 1]
```diff
@@ -225,6 +230,28 @@ def extract_sender_information(record):
...
+        if t['category'] == 'credit':
+            balance += t['amount']
+        elif t['category'] == 'debit':
+            balance -= t['amount']
+
+    try:
+        conn.balances.post({'date': date.isoformat(),
+                            'closing_balance': balance})
+    except HttpClientError:
+        # this is probably fine, just means that balance exists
```
Comment (ushkarev) [inline mtp_transaction_uploader/upload.py]: shouldn't we log this though? also, would it be a good idea to get the api to accept an overwrite for a date? or would that screw up reconciliation?

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Pull request merged**